pwndbg/auxv.py

```diff
@@ -105,11 +121,11 @@ def __str__(self) -> str:
 
 
 @pwndbg.lib.cache.cache_until("objfile", "start")
-def get() -> AUXV:
-    return use_info_auxv() or walk_stack() or AUXV()
+def get() -> AUXVProtocol:
+    return use_info_auxv() or walk_stack() or Foo()  # AUXV()
```

Comment:
Foo?

Reply:
I was testing that I would get a type error if the class didn't match the protocol, forgot to revert this part :P